src/app/shared/components/color-picker/color-picker.component.scss

```diff
@@ -0,0 +1,64 @@
+:host {
+  &.ng-invalid {
+    &.ng-touched {
+      input {
+        border-color: var(--danger-color);
+      }
+    }
+  }
+
+  &.ng-valid,
+  &.ng-untouched {
+    sofie-error-message {
+      display: none;
+    }
+  }
+}
+
+.color-picker {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  padding: 0.3rem 0.3rem;
+  font-weight: lighter;
+  font-size: 0.85rem;
+  font-family: Roboto, Arial, sans-serif;
+  border-radius: 4px;
+  cursor: pointer;
+  border: 1px solid var(--gray-color);
+
+  sofie-icon-button {
+    position: absolute;
+    right: 10px;
+    top: 8px;
+  }
+}
+
+.color-box {
+  padding: 0.2rem 0.5rem;
+  width: 50px;
+  height: 22px;
+  border-radius: 4px;
+}
```

Comment:
Setting both padding along with height and width seems unnecessary.
The mix of px and rem also makes it scale not ideally if the font-size changes.
I would suggest to set the height with rem and set the aspect ratio to something like 16/9, e.g.:
```scss
.color-box {
  height: 1.5rem;
  aspect-ratio: 16 / 9;
  border-radius: 4px;
}
```
The border radius is kept as pixels since, other border radii uses pixels as well.

Reply:
Updated